jekyll-assets/_layouts/docs.html

```diff
@@ -11,19 +11,19 @@
         <div id="toc-container">
           <div id="toc-inner">
             <div id="docs-header">
-              <h1 id="docs-header-title">
+              <span id="docs-header-title">
```

Comment:
Is this a semantic change to [avoid using more than one `<h1>` on a page](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page)? Given it _is_ a heading, should we use an `<h2>`, etc. instead?